201-machine-learning-advanced/azuredeploy.json

```diff
@@ -401,8 +401,7 @@
               "type": "Microsoft.Network/virtualNetworks/subnets",
               "apiVersion": "2019-09-01",
               "name": "[concat(parameters('vnetName'), '/', parameters('subnetName'))]",
-              "properties": "[if(and(equals(parameters('subnetOption'), 'existing'), not(equals(parameters('privateEndpointType'), 'none'))),
-                              union(reference(variables('subnet'), '2019-09-01'), variables('subnetPolicyForPE')), json('null'))]"
+              "properties": "[if(and(equals(parameters('subnetOption'), 'existing'), not(equals(parameters('privateEndpointType'), 'none'))), union(reference(variables('subnet'), '2019-09-01'), variables('subnetPolicyForPE')), json('null'))]"
```

Comment:
this seems to make the code less readable - why the change?

Reply:
@bmoore-msft When user copy and past the whole file into "ARM Template" in Azure portal, it throws validation error.